.github/workflows/build.yml

```diff
@@ -164,7 +164,7 @@ jobs:
         # Check if there are any empty files under ./site-docs/generated, If any empty files are found, print an error 
         # message and list the empty files
         run: |
-          tar zxvf core/build/distributions/kafka_2.13-$(./gradlew properties | grep version: | awk '{print $NF}' | head -n 1)-site-docs.tgz
+          tar zxvf distribution/build/distributions/kafka_2.13-$(./gradlew properties | grep version: | awk '{print $NF}' | head -n 1)-site-docs.tgz
```

Comment:
Changing the path may break downstream CI or workflows. Could you please provide more details about this change?